job-dsls/jobs/srcclr_scan_seed.groovy

```diff
@@ -0,0 +1,117 @@
+import org.kie.jenkins.jobdsl.Constants
+import org.kie.jenkins.jobdsl.templates.BasicJob
+
+String jobDescription = "Job responsible for SourceClear verification"
+
+def jobDefinition = job("srcclr-scan") {
+
+
+    parameters {
+        choiceParam('SCAN_TYPE', ['scm', 'binary'])
+        stringParam('URL','')
+        stringParam('VERSION', '')
+        stringParam('PACKAGE','')
+        stringParam('NAME', '')
+        stringParam('MVNPARAMS', '')
+        choiceParam('PROCESSOR_TYPE', ['cve', 'cvss'])
+        booleanParam('RECURSIVE', false)
+        booleanParam('DEBUGGING', false)
+        booleanParam('TRACING', false)
+        stringParam('SCMVERSION', '')
+        stringParam('THRESHOLD', '1','Threshold from 1 to 10 for cvss processor')
+    }
+
+
+    environmentVariables{
+        groovy('''
+          def map = [:]
+          
+          if ( "${RECURSIVE}" == "true" ) {
+            
+            map.put("RECURSE"," --recursive ")
+            
+          } else {
+            
+            map.put("RECURSE","")
+          
+          }
+          
+          if ( "${DEBUGGING}" == "true" ) {
+            map.put("DEBUG"," -d ")
+          } else {
+          
+            map.put("DEBUG","")
+          
+          }
+          if ( "${TRACING}" == "true" ) {
+            
+            map.put("TRACE"," --trace ")
+          
+          } else {
+            
+            map.put("TRACE","")
+            
+          }
+          if ( "${MVNPARAMS}" != "" )
+          {
+          
+            map.put("MVNPARAMETER","--maven-param=\\"${MVNPARAMS}\\"")
+            
+          } else {
+          
+            map.put("MVNPARAMETER", "")
+            
+          }
+          
+          if ( "${SCAN_TYPE}" == "scm" ) {
```

Comment:
The same here
```
map.put("SCMVERSIONPARAM", Boolean.valueOf(SCAN_TYPE) ? " --ref=${SCMVERSION}"" : "")
```

Reply:
Same here. Boolean.valueOf("scm) is false.

Reply:
yes the same as before
`map.put("SCMVERSIONPARAM", "${SCAN_TYPE}" != "" ? " --ref=${SCMVERSION}" : "")`